transactor/src/main/scala/io/mediachain/transactor/DatastoreRpcService.scala

```diff
@@ -0,0 +1,58 @@
+package io.mediachain.transactor
+
+// this is placed in io.mediachain.transactor to accompany the rest of the programs
+object DatastoreRpcService {
+  import io.mediachain.datastore.{DynamoDatastore, DatastoreService}
+  import io.mediachain.util.Properties
+  import io.grpc.Server
+  import scala.concurrent.ExecutionContext
+  import java.util.concurrent.Executors
+  import org.slf4j.LoggerFactory
+  
+  val logger = LoggerFactory.getLogger(RpcService.getClass)
+  
+  def main(args: Array[String]) {
+    if (args.length != 1) {
+      println("arguments: config-file")
```

Comment:
it's so insane to me that there's not a good, standard way to specify args, generate usage, etc in jvm land